docs/Security/Protected Services/index.md

```diff
@@ -0,0 +1,32 @@
+# Protected Services
+
+#### Related Evolution Proposals
+- [0317: SDL Protocol Security Specification](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0317-sdl-protocol-security-specification.md)
+
+## 1. Overview
+
+Secured services in SDL are established using a TLS handshake, the TLS handshake process is defined by TLS and is not part of the SDL protocol.

```

Comment:
Might be worth linking out to the diagram used in the protocol spec as well.

```
|||
TLS Handshake Process
![TLS Handshake activity diagram](https://user-images.githubusercontent.com/5848997/122258220-cb8b7100-ce9e-11eb-9b2a-a6194b0d1b68.png)
|||
```

Reply:
@jacobkeeler good idea, added in https://github.com/smartdevicelink/sdl_overview_guides/pull/80/commits/250cf04e5de7bbf6c19b72d99a91c3a7c0892463